src/layouts/default.html.handlebars

```diff
@@ -36,11 +38,10 @@
                 <div class="fl-panelBar">
                     <span class="fl-prefsEditor-buttons">
                         <button id="reset" class="flc-prefsEditor-reset fl-prefsEditor-reset"><span class="fl-icon-undo"></span> Reset</button>
-                        <button id="show-hide" class="flc-slidingPanel-toggleButton fl-prefsEditor-showHide"> Show/Hide</button>
+                        <button id="show-hide" class="flc-slidingPanel-toggleButton fl-prefsEditor-showHide">+ Show Display Preferences</button>
```

Comment:
This change is necessary to avoid the appalling jank as the UIO initialiser replaces the button text very late into initialisation of the page. Hopefully we can continue to fix the other sources of jank bugs such as https://issues.fluidproject.org/browse/FLUID-5481 since they give our docs site an amateurish appearance. I don't believe that any of our typical docs users will find it valuable to tell "at a glance" whether UIO initialized or not : P